grpc/cpp-diatheke/diatheke_client.h

```diff
@@ -73,6 +74,16 @@ class Client
     cobaltspeech::diatheke::SessionOutput
     createSession(const std::string &modelID);
 
+    /*
+     * Create a new Diatheke session using the specified
+     * model ID and custom wakeword. The wakeword will only
+     * have an effect if the model has wakeword detection
+     * enabled.
```

Comment:
Is there any checks we should do on the wakeword, depending on the model info?  Or should this client be dumb and just hope for the best?

Reply:
I don't think we need to do any checks on the client side. If the model uses a wakeword, we will attempt to use the custom one. If not, it is ignored. If a client application tries to use a phrase that is terrible for a wakeword and it doesn't end up working so well, we can take the opportunity to educate them that not all wakewords are created equal. Or if it turns out to be a significant issue, we can always do some checks on the server side (I think my preference would be for this kind of check to be on the server side regardless - the server has much more information than the client and usually more resources available to effectively do that kind of check).